docs/module_usage/instructions/benchmark.en.md

```diff
@@ -112,7 +112,7 @@ After enabling the benchmark feature, the benchmark results will be automaticall
             <li>model inference time (<code>Inference</code>)</li>
             <li>postprocessing time (<code>Postprocessing</code>)</li>
             <li>core time (<code>Core</code>, i.e., Preprocessing + Inference + Postprocessing)</li>
-            <li>other time (<code>Other</code>)</li>
+            <li>other time (<code>Other</code>, such as the observation and statistical times generated when enabling the benchmark feature)</li>
```

Comment:
, e.g., the time taken to run the code that orchestrates the operations, and the extra overhead introduced by the benchmark feature.